src/main.ts

```diff
@@ -25,21 +25,22 @@ if (!app.requestSingleInstanceLock()) {
 } else {
     // Your data now belongs to CCP
     crashReporter.start({uploadToServer: false});
-    if (process.env.USE_WAYLAND == "0") {
-        console.log("Wayland patches disabled.");
-    } else {
-        if (process.platform == "linux") {
-            if (process.env.XDG_SESSION_TYPE == "wayland") {
-                console.log("Wayland specific patches applied.");
-                app.commandLine.appendSwitch("ozone-platform=wayland");
-                if (process.env.XDG_CURRENT_DESKTOP == "GNOME") {
-                    app.commandLine.appendSwitch("enable-features=UseOzonePlatform,WaylandWindowDecorations");
-                } else {
-                    app.commandLine.appendSwitch("enable-features=UseOzonePlatform");
-                }
+    if (process.platform == "linux") {
```

Comment:
Why not just do 
`
if (process.platform === "linux" && process.env.XDG_SESSION_TYPE == "wayland") {...}`

Reply:
Also, what's with the `===`? Wouldn't just `==` work?

Reply:
== is loose equality ("0" == 0), === is strict ("0" !== 0) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
we might not want that then, assuming === makes Wayland and wayland seem different (if == doesn't already do that).

Reply:
=== doesn't do that either, if you want case insensitive comparison you should call .toLowerCase() on both items

Reply:
It's not too much of an issue anyways, since AFAIK most (or all) set `wayland` and not `Wayland`.

Reply:
imo you should do the toLowerCase() so there is no "WhY NO WorKy"

Reply:
Alright, I think that's it.

Reply:
wait crap I think I did it wrong... could you write me a snippet or another review suggestion to do it?